gapic/src/main/com/google/gapid/widgets/TabComposite.java

```diff
@@ -1211,4 +1530,8 @@ public void close() {
       shell.dispose();
     }
   }
+
+  public interface TabManager {
+    public void closeTab(Object id);
```

Comment:
Instead, add `onTabClosed(TabInfo tab)` in the existing `Listener`.

Reply:
Thanks, done, I also prefer that solution. I also added onTabCreated to be consistent.